src/style/_hero-img-h1.scss

```diff
@@ -0,0 +1,112 @@
+
+.section__hero-img__H1{
+    background: url('/images/headerimage_mobile.jpg');
+    width: 100vw;
+    height: 163vw;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center center;
+}
+
+.h1__section__div{
+    display: flex;
+    position: relative;
+}
+
+.h1__wheatly{
+    font-family: $h1Font; 
+    font-weight: 300;
+    color: $darkTeal;
+    font-size: 3.7rem;
+    text-transform: uppercase;
+    letter-spacing: 0.2rem;
+    margin-top: 140px;
+    margin-bottom: 20px;
+    margin-left: 24%;
+    position: relative;
+}
+
+.h1__span {
+    color: black;
+    font-size: 6.5rem;
+    position: absolute;
+    left: -75px;
+    bottom: 8px;
+}
+
+.section__hero-img__H1__hidden-p{
+    visibility: hidden;
+}
+
+@media screen and (min-width: 744px) {
```

Comment:
samma här med breakpoint. Importera in och använd variablerna från varaiables.scss filen.

Reply:
Fixat.